microsoft-dynamics-365/2.6/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -0,0 +1,1079 @@
+
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Logical Name a| String |  Logical name of the entity. It is always the schema name in lower case. |  | x
+| Attributes a| Object |  Attributes of the entity to create as a map. |  `#[payload]` |
+| Encoding a| a| Body encoding.
+
+* If no value is specified, set the Mule default encoding.
+
+* If a value is specified, set the `-M-Dmule.encoding` VM argument to the same value to avoid an encoding mismatch. | |
+
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+
+[%autowidth.spread]
+|===
+|Type |String
+|===
+
+==== Associated Configurations
+
+* <<dynamics-config>>
+
+==== Throws
+
+* DYNAMICS:CONNECTIVITY
+* DYNAMICS:INVALID_PAGE_SIZE
+* DYNAMICS:NOT_FOUND
+* DYNAMICS:OPERATION_FAILED
+* DYNAMICS:PARSE_ERROR
+* DYNAMICS:RETRY_EXHAUSTED
+* DYNAMICS:TIMEOUT
+* DYNAMICS:UNAUTHORIZED
+* DYNAMICS:UNKNOWN
+
+
+[[createMultiple]]
+=== Create Multiple
+
+`<dynamics:create-multiple>`
+
+Creates multiple entities.
+
+* Create New Entity - `+https://msdn.microsoft.com/en-us/library/gg328090.aspx+`
+* Associate Entity Upon Creation - `+https://msdn.microsoft.com/en-us/library/mt607875.aspx#Anchor_4+`
+* Batch Request - `+https://msdn.microsoft.com/en-us/library/mt607719.aspx+`
+
+
+
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Logical Name a| String |  Logical name of the entity. It is always the schema name in lower case. |  | x
+| Entities Attributes a| Array of Object |  List of the entities' attributes to create as a list of maps. |  `#[payload]` |
+| Use Single Transaction a| Boolean |  Use a single transaction for creation. If the entity creation fails, the transaction is rolled back. |  `false` |
+| Continue On Error a| Boolean |  Continue in case of error. The default value of `false` stops creating records after one create fails.
+
+You can use *Continue On Error* in conjunction with *Use Single Transaction*. Suppose you want to execute an operation with a payload of 10 records. This payload contains one broken record that will fail. This broken record is record number 5. The other nine records are certain to succeed. The following table shows the output depending on the values of *Continue On Error* and *Use Single Transaction*:
```

Comment:
```suggestion
You can use *Continue On Error* in conjunction with *Use Single Transaction*. Suppose you want to execute an operation with a payload of 10 records. This payload contains one broken record, number 5, that fails. The other 9 records succeed. The following table shows the output, which depends on the values of *Continue On Error* and *Use Single Transaction*:
```

Reply:
Once you use a number instead of text for a number, you can use all numbers for consistency in a paragraph.